meson.build

```diff
@@ -0,0 +1,222 @@
+project('libass', 'c', license: 'ISC',
+        meson_version: '>= 0.55.0',
```

Comment:
Not too surprising; maintaining compatibility with old versions wasn't a particular priority for me. I'm always fetching the latest version, and presumably on Windows/MacOS this will be the case for most users.